### ISSUE TITLE
After upgrading from 8.2.3 to 9.0.0 no files, contacts, or calendar data shown in web interface

### ISSUE THREAD

**Issue by mstuehrenberg** [owncloud/core#23014]
### Steps to reproduce
1. Updated owncloud 8.2.2 to 8.2.3 according to docs: 
- set maintenance:mode on
-  cp owncloud-dir to owncloud-8.2.2 (data-dir is external) and mysqldump database
- stop apache
- mv owncloud-dir to owncloud.old
- download owncloud 8.2.3 and extract to webroot/owncloud
- copy config.php from owncloud-8.2.2 and apps/ownNote to owncloud
- start apache
- run occ upgrade
- deactivate maintenance mode
- log in to web interface as admin and active only third-party plugin ownNote
Result: works, everything is fine
2. Update owncloud 8.2.3 to 9.0 according to docs (same steps as above)

### Expected behaviour
owncloud 9.0.0 is installed and all files, contacts and calendars are present in the web interface

### Actual behaviour
owncloud 9.0.0 is installed but there are neither files nor contacts or calendars present in the web interface. Both user accounts (admin and personal account) are present and PHPMyAdmin shows the old and new tables (including content) in the owncloud DB but the web interface is empty.

I did a php console.php files:scan --all, files were shown correctly and were present in oc-data directory on the server.

Note: I did not try to sync any other client than the web interface, so it may be that the update went smoothly but the display of the web interface is the only problem.

### Server configuration
**Operating system**: Linux Busy Box under QNAP OS 4.2 (x86)

**Web server:**
Apache 2.x

**Database:**
MariaDB 5.5.44.002

**PHP version:**
5.5.31

**ownCloud version:** (see ownCloud admin page)
9.0.0

**Updated from an older ownCloud or fresh install:**
updated from 8.2.3 after upgrading from 8.2.2

**Where did you install ownCloud from:**
official oc homepage https://download.owncloud.org/community/owncloud-9.0.0.tar.bz2

**Signing status (ownCloud 9.0 and above):**

```
Not possible anymore, since I have already restored to 8.2.3
```

**List of activated apps:**
OwnNote

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder
```

**The content of config/config.php:**
```
sudo -u www-data php occ config:list system
from within your ownCloud installation folder

see above
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
no

**Are you using encryption:** yes/no
no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

### Client configuration
**Browser:**
Google Chrome 49.0.2623.87

**Operating system:**
OS X 10.11.2

**Comment by nickvergessen** [comment]
You missed to provide the necessary information: config and app list.

**Comment by mstuehrenberg** [comment]
config.php:

<?php
$CONFIG = array (
  'instanceid' => '<removed>',
  'passwordsalt' => '<removed>',
  'secret' => '<removed>',
  'trusted_domains' => 
  array (
    0 => '<removed>',
    1 => '<removed>',
    2 => '<removed>',
  ),
  'datadirectory' => '/share/MD0_DATA/Qweb/owncloud-data',
  'overwrite.cli.url' => '<removed>',
  'dbtype' => 'mysql',
  'version' => '8.2.3.2',
  'installed' => true,
  'loglevel' => 1,
  'theme' => '',
  'maintenance' => true,
  'dbname' => '<removed>',
  'dbhost' => 'localhost:3306',
  'dbuser' => '<removed>',
  'dbpassword' => '<removed>',
  'trashbin_retention_obligation' => 'auto',
);

Installed (active) apps are contact, calendar and ownNote

**Comment by webwude** [comment]
Almost the same here:

I tried to upgrade from 8.2.3 to 9. I am using a shared webspace environament with ssl proxy in Germany (all-inkl.com) which used to work just fine.
My configuration already consists of seperate data and log directory, so I just saved my old oc instance and installed the zip from scratch.

1. I copied and unpacked the zip file and copied my old config.php into the config folder
2. I opened the website and started the upgrade. 

Error: internal server error - inside of my htaccess i had to remove a "RewriteBase" line, otherwise the .htaccess will not be accepted.

First login: error message: your webserver is not configured for data synchronisation - webdav is malfunctioning.
Directory is not available.
Log:
Error	PHP require_once(/<rootdir>/owncloud/apps/files/appinfo/remote.php): failed to open stream: No such file or directory at /<rootdir>/owncloud/remote.php#138

Error	PHP	require_once(): Failed opening required '/<rootdir>/owncloud/apps/files/appinfo/remote.php' (include_path='/<rootdir>/owncloud/3rdparty/pear/console_getopt:/<rootdir>/owncloud/3rdparty/pear/pear_exception:/<rootdir>/owncloud/3rdparty/pear/pear-core-minimal/src:/<rootdir>/owncloud/3rdparty/pear/archive_tar:/<rootdir>/owncloud/3rdparty/phpseclib/phpseclib/phpseclib:/<rootdir>/owncloud/lib/private:/<rootdir>/owncloud/config:/<rootdir>/owncloud/3rdparty:/<rootdir>/owncloud/apps:.:/usr/share/php:..:/<rootdir>/owncloud') at /<rootdir>/owncloud/remote.php#138

No access to files anymore...
Any ideas?

-- Update: calendar and contacts are gone as well or not accessible - I will go back to 8.2.3 - not quite what I have expected...﻿

Config file:

<?php
$CONFIG = array (
  'instanceid' => 'xxx',
  'passwordsalt' => 'xxx',
  'secret' => 'xxx',
  'trusted_domains' => 
  array (
    0 => 'mydomain',
  ),
  'datadirectory' => '<rootdir>/owncloud_data',
  'dbtype' => 'mysql',
  'version' => '9.0.0.19',
  'dbname' => 'xxx',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'xxx',
  'dbpassword' => 'xxx',
  'installed' => true,
  'maxZipInputSize' => 0,
  'allowZipDownload' => true,
  'overwritehost' => 'ssl-account.com',
  'overwrite.cli.url' => 'http://mydomain',
  'overwriteprotocol' => 'https',
  'overwritewebroot' => '/mydomain',
  'overwritecondaddr' => '^myip$',
  'forcessl' => 'true',
  'mail_from_address' => 'owncloud',
  'mail_smtpmode' => 'php',
  'mail_domain' => 'mydomain',
  'defaultapp' => 'files, activity',
  'logdateformat' => 'F d, Y H:i:s',
  'logtimezone' => 'Europe/Berlin',
  'loglevel' => 3,
  'log_rotate_size' => 10485760,
  'tempdirectory' => '<rootdir>/tmp/',
  'logfile' => '<rootdir>/owncloud_logs/owncloud.txt',
  'check_for_working_htaccess' => true,
  'trashbin_retention_obligation' => 'auto',
  'appstore.experimental.enabled' => true,
  'activity_expire_days' => 7,
  'maintenance' => false,
  'ldapIgnoreNamingRules' => false,
  'theme' => '',
);

 if(array_key_exists('HTTP_X_FORWARDED_SERVER', $_SERVER) && $_SERVER['HTTP_X_FORWARDED_SERVER'] == 'ssl-account.com')
{
  $CONFIG['overwritehost'] = 'ssl-account.com';
  $CONFIG['overwriteprotocol'] = 'https';
  $CONFIG['overwritewebroot'] = str_replace('http:/', '', $CONFIG['overwrite.cli.url']);
  $CONFIG['overwritecondaddr'] = '^'.str_replace('.','\.',$_SERVER['REMOTE_ADDR']).'$';
}; 

Apps all disabled during upgrade - afterwards contacts and calendar enabled...

**Comment by globalcow** [comment]
I had a similar problem. My installation is on shared webspace server. I had to remove 'Options -MultiViews' from the .htaccess file as my provider is not supporting Options directives in htaccess. Perhaps it is a similar problem for you.

**Comment by cubical23** [comment]
I had problems with calendar, calendars were not visible with current IE (especially the calendar list on the left), firefox worked as expected.

**Comment by RealRancor** [comment]
Yes, IE and EDGE are currently not supported by the calendar app as discussed here: https://github.com/owncloud/calendar/issues/309

**Comment by cubical23** [comment]
thanks for your reply. that's sad, especially for use with unexperienced users.

**Comment by PVince81** [comment]
@georgehrke is this still an issue ?

**Comment by georgehrke** [comment]
There were plenty of fixes in 9.0.1 to fix migration of contacts and calendars.
If not, speak up please.
